frontend/src/components/LoginPage/LoginPage.css

```
@@ -0,0 +1,141 @@
+#loginBox {
+    height: 450px;
+    width: 350px;
+    background: #FEFEFE;
+    border: 0.5px solid #C4C4C4;
+    margin: auto;
+    margin-top: 18vh;
```

```suggestion
    margin-top: 50vh;
    transform: translateY(-50%);
```
Centering the box on the screen